docker-compose.yml

```
@@ -45,6 +45,7 @@ services:
     environment:
       TYPESENSE_DATA_DIR: ${TYPESENSE_DATA_DIR}
       TYPESENSE_API_KEY: ${TYPESENSE_API_KEY}
+      TYPESENSE_ENABLE_CORS: ${TYPESENSE_ENABLE_CORS}
```

```suggestion
      TYPESENSE_ENABLE_CORS: true
```
Ce sera toujours vrai donc autant le mettre ici directement